File: subprojects/shell/git/src/main/kotlin/dotfiles/shell/git/personal.kt
package dotfiles.shell.git

import io.mkobit.git.config.Commit
import io.mkobit.git.config.Gpg
import io.mkobit.git.config.Section
import io.mkobit.git.config.User

internal fun personalGitConfig(): List<Section> = listOf(
  Commit(
    gpgSign = true
  ),
  Gpg(
    program = "gpg"
  ),
  User(
    email = "mkobit@gmail.com",
    userName = "Mike Kobit",
    signingKey = "1698254E135D7ADE!"
  )
)

File: homedir/git/personal.py
from typing import List
from src.dotfiles.git.config.commit import Commit
from src.dotfiles.git.config.gpg import Gpg
from src.dotfiles.git.config.section import Section
from src.dotfiles.git.config.user import User


def personal_git_config() -> List[Section]:
    personal = [
        Commit(gpg_sign=True),
        Gpg(program="gpg"),
        User(
            email="mkobit@gmail.com",
            user_name="Mike Kobit",
            signing_key="1698254E135D7ADE!",
        ),
    ]
    return personal
